sport/test/tools/MatchListFeatureTools.scala

```diff
@@ -45,10 +46,11 @@ trait MatchListFeatureTools extends Matchers {
     )
   }
   protected def scrollToElementAndClick(selector: String, browser: TestBrowser): Unit = {
-    val element = browser.$(selector).first
-    val builder = new Actions(browser.webDriver)
-    builder.moveToElement(element.getElement)
-    builder.click()
-    builder.build().perform()
+    val webDriver = browser.webDriver
+    val element = webDriver.findElement(By.cssSelector(selector))
+    new Actions(webDriver)
+      .moveToElement(element)
+      .click()
+      .perform()
```

Comment:
After upgrading to Play 2.9 we have to change `org.fluentlenium` imports to `io.fluentlenium` (see also [migration guide](https://www.playframework.com/documentation/2.9.x/Migration29#Namespace-Change:-org.fluentlenium-to-io.fluentlenium)). This change also brings a new version of `selenium-api` (4-14.1 from 3.141.59), so we have to slightly modify our test code. This fixes a problem of tests timeouting when the method `scrollToElementAndClick` gets called, e.g. https://github.com/guardian/frontend/blob/main/sport/test/FixturesFeatureTest.scala#L44